script/tool/test/version_check_test.dart

```diff
@@ -196,6 +196,73 @@ void main() {
       expect(gitDirCommands[2].join(' '),
           equals('show HEAD:packages/plugin_platform_interface/pubspec.yaml'));
     });
+
+     test('Throws if first line in change log is empty', () async{
```

Comment:
Why do we want to do this? Isn't an empty first line a harmless cosmetic thing that we should just ignore?